tests/strategies/parse/test_image.py

```diff
@@ -0,0 +1,76 @@
+"""Test the image formats in the image parse strategy."""
+from pathlib import Path
+from typing import Tuple
+
+import pytest
+
+
+def get_cropping_values(ext: str) -> Tuple[int]:
+    """Return the cropping values for the test."""
+    croppings = {
+        "gif": (200, 300, 900, 700),
+        "jpeg": (200, 300, 900, 700),
+        "jpg": (200, 300, 900, 700),
+        "jp2": (1000, 1000, 2000, 2000),
+        "png": (100, 50, 450, 300),
+        "tiff": (100, 50, 450, 300),
+    }
+    return croppings[ext]
+
+
+def get_orig_path(ext: str) -> Path:
+    """Return the path to the original image in the test."""
+    dirs = Path(__file__).resolve().parents
+    origs = {
+        "gif": dirs[0] / "sample_1280_853.gif",
+        "jpeg": dirs[1] / "sample_1280_853.jpeg",
+        "jpg": dirs[1] / "sample_1280_853.jpg",
+        "jp2": dirs[0] / "sample1.jp2",
+        "png": dirs[0] / "sample_640_426.png",
+        "tiff": dirs[0] / "sample_640_426.tiff",
+    }
+    return origs[ext]
+
+
+def get_target_path(ext: str) -> Path:
+    """Return the path to the target contents for the cropped image
+    in the test.
+    """
+    dirs = Path(__file__).resolve().parents
+    targets = {
+        "gif": dirs[0] / "sample_700_400.gif",
+        "jpeg": dirs[1] / "sample_700_400.jpeg",
+        "jpg": dirs[1] / "sample_700_400.jpeg",
+        "jp2": dirs[0] / "sample1_1000_1000.jp2",
+        "png": dirs[0] / "sample_350_250.png",
+        "tiff": dirs[0] / "sample_350_250.tiff",
+    }
+    return targets[ext]
```

Comment:
There's no difference between these functions as I see it?
Also, why are some files under the second parent directory and others in the first?

Reply:
`get_orig_path()` gets the input files, while `get_target_path()` gets the targets to compare the output to.

Reply:
As written in a previous commit (one of the first in this PR), some of the files were put in the `strategies` folder, to be available to other (non-parser) tests as well.